src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java

```diff
@@ -416,22 +417,37 @@ private int recordExitStatus(FilePath workspace, int code) throws IOException, I
         private static final long serialVersionUID = 7599995179651071957L;
         private final OsType os;
         private final String binaryPath;
+        private String architecture;
```

Comment:
```suggestion
        private final String architecture;
```

Reply:
made final in dd9324a8e1e9ed44b845c57cd9d3ddac2ea68939